store/snackbar.ts

```diff
@@ -1,18 +1,20 @@
 import { ActionTree, MutationTree } from 'vuex';
 
+type colors = 'error' | 'warning';
```

Comment:
Technically, this can be any member of `VuetifyThemeVariant` (I believe) or anything from https://vuetifyjs.com/en/styles/colors/#javascript-color-pack

Reply:
I've removed these changes. It seems, to allow for any colors.